grafana-plugin/src/components/Avatar/Avatar.test.tsx

```diff
@@ -1,12 +1,9 @@
 import React from 'react';
 
-import { describe, expect, test } from '@jest/globals';
```

Comment:
this is automatically "injected" into test files, not need to import this. Additionally, `@testing-library/jest-dom` is now imported in `jest.setup.ts` so there's no need to import this in every test file.